document/.gitignore

```diff
@@ -0,0 +1,2 @@
+_build
```

Comment:
Any particular reason why this is _build as opposed to something like out or build (no underscore)?